RELEASE-NOTES.md

```diff
@@ -2,6 +2,16 @@
 
 ## PyMC3 3.9.x (on deck)
 
+### Maintenance
+
+### Documentation
+
+### New features
+- `sample_posterior_predictive_w` can now feed on `xarray.Dataset` - e.g. from `InferenceData.posterior`. (see [#4042](https://github.com/pymc-devs/pymc3/pull/4042))
+
+
+## PyMC3 3.9.3 (11 August 2020)
+
```

Comment:
I updated the release notes in an earlier PR: can you merge master into this branch and then add a line for this PR please?